llvm/lib/CheerpUtils/Utility.cpp

```diff
@@ -285,6 +285,11 @@ bool InlineableCache::isInlineableImpl(const Instruction& I)
 	}
 	else if(I.getOpcode()==Instruction::BitCast)
 	{
+		if (!I.getType()->isPointerTy())
+		{
+			assert(I.getType()->isVectorTy());
+			return true;
```

Comment:
This I need to actually think about, unsure whether should be return !hasMoreThan1Use. Reasoning, BitCast on vector are no-op, but unsure if it's properly tracked by users

Reply:
Done, always returning true did indeed introduce multiple redundant statements